index.js

```diff
@@ -1,100 +1,122 @@
+"use strict";
+
 var _ = require('underscore');
+var cnchars = require('cn-chars'); // convert between traditional and simplified
 var path = require('path');
+var pinyin = require('prettify-pinyin'); // prettify the pinyin default (letters + numbers) in CC-CEDICT
 var Sequelize = require('sequelize');
+
 var sequelize = new Sequelize(null, null, null, {
   dialect: 'sqlite',
-  storage: path.join(__dirname, './db/cc-cedict.sqlite')
+  storage: path.join(__dirname, './db/', 'cc-cedict.sqlite')
 });
 
-var Word = sequelize.define('Word', {
-  traditional: Sequelize.STRING,
-  simplified: Sequelize.STRING,
-  pronunciation: Sequelize.STRING,
-  definitions: Sequelize.STRING
+var Word = sequelize.define('word', {
+  traditional: {
+    type: Sequelize.STRING
+  },
+  simplified: {
+    type: Sequelize.STRING
+  },
+  pronunciation: {
+    type: Sequelize.STRING
+  },
+  definitions: {
+    type: Sequelize.STRING
+  }
 });
 
-// convert between traditional and simplified
-var cnchars = require('cn-chars');
-// prettify the pinyin default (letters + numbers) in CC-CEDICT
-var pinyin = require('prettify-pinyin');
-
-module.exports.searchByChinese = function(str, cb){
+module.exports.searchByChinese = function (str, cb) {
```

Comment:
There should not be any whitespace in between the function keyword and opening parens, as well as after the closing parens after parameters, e.g.:

function(a, b){
  // do stuff
}